pkg/cmd/repo/create/create.go

```diff
@@ -681,3 +682,8 @@ func getVisibility() (string, error) {
 
 	return strings.ToUpper(answer.RepoVisibility), nil
 }
+
+// normalizeRepoName takes in the repo name the user inputted and normalizes it using the same logic as GitHub (GitHub.com/new)
+func normalizeRepoName(repoName string) string {
+	return strings.TrimSuffix(regexp.MustCompile(`[^\w.-]+`).ReplaceAllString(repoName, "-"), ".git")
```

Comment:
Underscore `_` is also an allowed character

Reply:
https://regex101.com/r/Sl6WtS/1
https://pkg.go.dev/regexp/syntax

I think `\w` includes `_`? I added `_` to the regex just to be sure, but I supplied a test golang regex link above as well as the docs.